src/CMakeLists.txt

```diff
@@ -7,6 +7,8 @@
 ###############################################################################
 
 set(libsrc
+  string.c
+  spawn-zos.cc
```

Comment:
Although it is redundant, source files have a zos- prefix.

Also, it may make sense to add string.c contents to zos-char-util.cc.